secretcache/cacheItem.go

```diff
@@ -129,6 +129,12 @@ func (ci *secretCacheItem) getVersion(versionStage string) (*cacheVersion, bool)
 	return secretCacheVersion, true
 }
 
+// refresh the cached object on demand
+func (ci *secretCacheItem) refreshNow(ctx context.Context) {
```

Comment:
It's good to have this here to allow immediate refresh of the cache item, but can we also add something similar in the cache itself? See [the java implementation](https://github.com/aws/aws-secretsmanager-caching-java/blob/2d42f76af59568ec2151e000a27bd12f78a03c77/src/main/java/com/amazonaws/secretsmanager/caching/SecretCache.java#L162) for some details.

Reply:
Added this in new commit